### PR TITLE
Just simplify some point.

### PR DESCRIPTION
**Pull request by hotchemi**
## Overview

- Just simplify some point.

---

I hereby agree to the terms of the JUnit Contributor License Agreement.

### CODE REVIEW

File: junit-console/src/main/java/org/junit/gen5/console/ConsoleRunner.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public static void main(String... args) {
 	int execute(String... args) {
 		CommandLineOptions options = commandLineOptionsParser.parse(args);
 		ConsoleTask task = determineTask(options);
-		return consoleTaskExecutor.executeTask(task, out -> displayHelp(out));
+		return consoleTaskExecutor.executeTask(task, this::displayHelp);
```
Comment (marcphilipp) [inline junit-console/src/main/java/org/junit/gen5/console/ConsoleRunner.java]: :+1:

File: junit5-engine/src/main/java/org/junit/gen5/engine/junit5/discovery/DiscoverySelectorResolver.java [hunk 2]
```diff
@@ -59,15 +59,10 @@ public void resolveSelectors(EngineDiscoveryRequest request) {
...
-		request.getSelectorsByType(MethodSelector.class).forEach(selector -> {
-			resolveTestMethod(selector.getTestClass(), selector.getTestMethod());
-		});
-		request.getSelectorsByType(UniqueIdSelector.class).forEach(selector -> {
-			resolveUniqueId(selector.getUniqueId());
-		});
+		request.getSelectorsByType(ClassSelector.class).forEach(selector -> resolveTestClass(selector.getTestClass()));
+		request.getSelectorsByType(MethodSelector.class).forEach(
+			selector -> resolveTestMethod(selector.getTestClass(), selector.getTestMethod()));
+		request.getSelectorsByType(UniqueIdSelector.class).forEach(selector -> resolveUniqueId(selector.getUniqueId()));
```
Comment (marcphilipp) [inline junit5-engine/src/main/java/org/junit/gen5/engine/junit5/discovery/DiscoverySelectorResolver.java]: I did this on purpose for better readability (which is of course subjective). Can you please change it back?

File: surefire-junit5/src/main/java/org/junit/gen5/surefire/JUnitGen5Provider.java [hunk 3]
```diff
@@ -60,8 +60,7 @@ public RunResult invoke(Object forkTestSet)
 	private TestsToRun scanClasspath(Launcher launcher) {
 		TestsToRun scannedClasses = parameters.getScanResult().applyFilter(new TestPlanScannerFilter(launcher),
 			parameters.getTestClassLoader());
-		TestsToRun orderedClasses = parameters.getRunOrderCalculator().orderTestClasses(scannedClasses);
-		return orderedClasses;
+		return parameters.getRunOrderCalculator().orderTestClasses(scannedClasses);
```
Comment (marcphilipp) [inline surefire-junit5/src/main/java/org/junit/gen5/surefire/JUnitGen5Provider.java]: :+1:

File: junit5-engine/src/main/java/org/junit/gen5/engine/junit5/discovery/DiscoverySelectorResolver.java (hunk 2 above)
Comment (hotchemi) [inline junit5-engine/src/main/java/org/junit/gen5/engine/junit5/discovery/DiscoverySelectorResolver.java]: squashed.

**Comment by marcphilipp** [comment]
Thanks!

**Pull request merged**